samples/subsys/usb/uvc/app.overlay

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024 tinyVision.ai Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/usb/video.h>
+
+/ {
+	chosen {
+		zephyr,console = &usart3;
+		zephyr,shell-uart = &usart3;
```

Comment:
We need a generic sample that works with any board/USB controller, ideally with no cam connected by default, just streaming some test data.